devel/hs-transformers/Portfile

```diff
@@ -0,0 +1,20 @@
+PortSystem          1.0
+PortGroup           haskell 1.0
+
+haskell.setup       transformers 0.4.3.0
```

Comment:
I don't know fore sure (not super familiar with haskell ecosystem), but if I didn't install [transformers](https://hackage.haskell.org/package/transformers), ShellCheck installation will complaint about missing package.